Pod/Classes/URBNArrayDataSourceAdapter.m

```diff
@@ -93,6 +93,27 @@ - (void)appendItems:(NSArray *)newItems inSection:(NSInteger)section {
     }
 }
 
+- (void)appendSectionWithItems:(NSArray *)newItems {
+    
+    if ([self isSectioned]) {
```

Comment:
Also if you aren't going to perform an action lets throw an assert in there. So the consumer knows why it was a no-op and to stop doing it.